run.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Simple script to build things with the appropriate flags and see the Caliper output